docs/sources/mimir/manage/run-production-environment/production-tips/index.md

```diff
@@ -191,14 +191,16 @@ around 400MiB/s. For the same data gzip achieves a ratio between 6 and 8 with sp
 To configure gRPC compression, use the following CLI flags or their YAML equivalents. The accepted values are
 `snappy` and `gzip`. If you set the flag to an empty string (`''`), it explicitly disables compression.
 
-| CLI flag                                                    | YAML option                                                |
-| ----------------------------------------------------------- | ---------------------------------------------------------- |
-| `-query-frontend.grpc-client-config.grpc-compression`       | `alertmanager.alertmanager_client.grpc_compression`        |
-| `-query-scheduler.grpc-client-config.grpc-compression`      | `frontend.grpc_client_config.grpc_compression`             |
-| `-ruler.client.grpc-compression`                            | `frontend_worker.grpc_client_config.grpc_compression`      |
-| `-ruler.query-frontend.grpc-client-config.grpc-compression` | `ingester_client.grpc_client_config.grpc_compression`      |
-| `-alertmanager.alertmanager-client.grpc-compression`        | `query_scheduler.grpc_client_config.grpc_compression`      |
-| `-ingester.client.grpc-compression`                         | `ruler.query_frontend.grpc_client_config.grpc_compression` |
+| CLI flag                                                         | YAML option                                                |
+| ---------------------------------------------------------------- | ---------------------------------------------------------- |
+| `-query-frontend.grpc-client-config.grpc-compression`            | `alertmanager.alertmanager_client.grpc_compression`        |
+| `-query-scheduler.grpc-client-config.grpc-compression`           | `frontend.grpc_client_config.grpc_compression`             |
+| `-ruler.client.grpc-compression`                                 | `frontend_worker.grpc_client_config.grpc_compression`      |
+| `-ruler.query-frontend.grpc-client-config.grpc-compression` [^1] | `ingester_client.grpc_client_config.grpc_compression`      |
+| `-alertmanager.alertmanager-client.grpc-compression`             | `query_scheduler.grpc_client_config.grpc_compression`      |
+| `-ingester.client.grpc-compression`                              | `ruler.query_frontend.grpc_client_config.grpc_compression` |
+
+[^1] Only applicable when the ruler uses gRPC to communicate with the query-frontend. See [Remote ruler mode](../../../references/architecture/components/ruler/#remote-over-http-https).
```

Comment:
```suggestion
{{< admonition type="note" >}}
`-ruler.query-frontend.grpc-client-config.grpc-compression` is only applicable when the ruler uses gRPC to communicate with the query-frontend. Refer to [Remote ruler mode](../../../references/architecture/components/ruler/#remote-over-http-https).
{{< /admonition >}}
```

Reply:
Would you mind removing the footnote here and instead formatting it as a note? Footnotes can present accessibility challenges.